src/js/lib/monster.ui.js

```diff
@@ -494,12 +494,13 @@ define(function(require) {
 				}
 
 				var newTag = $(this).val(),
-					option = $('<option>').val(newTag).text(newTag);
+					option = $('<option>')
+						.val(newTag)
+						.text(newTag)
+						.prop('selected', true);
 
 				// Add the new option
 				$target.prepend(option);
```

Comment:
Seems like using `jquery#prepend` messes up the number of options selected:
1. add a new option by hitting enter
1. select a different option than the one just added
1. click on chosen to open the dropdown

=> the option selected in the dropdown is still the one added programmatically

Looks like adding the option with `jquery#append` does not produce this side effect but please test more use cases involving chosen's different options (multilpe selection for example).

Reply:
awesome, let me try this